network/init.sh

```diff
@@ -1,31 +1,89 @@
 #!/bin/bash
-DEV_MNEMONIC_1="alley afraid soup fall idea toss can goose become valve initial strong forward bright dish figure check leopard decide warfare hub unusual join cart"
-DEV_MNEMONIC_2="record gift you once hip style during joke field prize dust unique length more pencil transfer quit train device arrive energy sort steak upset"
 
-### Chain 1
-yes $DEV_MNEMONIC_1 | icad keys add val --recover --home ~/.demo-test-1
+# Configure predetermined settings
+# val1: cosmos1mjk79fjjgpplak5wq838w0yd982gzkyfrk07am
+# val2: cosmos17dtl0mjt3t77kpuhg2edqzjpszulwhgzuj9ljs
+BINARY=icad
```

Comment:
nit: I think all vars here should be upper case snake case. `CHAIN_DIR`

Reply:
I will fix them all following the convention.